python/cugraph/cugraph/tests/mg/test_mg_neighborhood_sampling.py

```diff
@@ -120,30 +121,43 @@ def test_mg_neighborhood_sampling_simple(dask_client, input_combo):
         right_on=[*input_df.columns[:2]])
     if len(result_nbr) != len(join):
         join2 = input_df.merge(
-            result_nbr, how='left', left_on=[*input_df.columns],
+            result_nbr, how='right', left_on=[*input_df.columns],
             right_on=[*result_nbr.columns])
+
         pd.set_option('display.max_rows', 500)
-        print('df1 = \n', input_df.sort_values([*input_df.columns]))
-        print('df2 = \n', result_nbr.sort_values(
+        # print('input_df = \n', input_df.sort_values([*input_df.columns]))
+        print('result_df = \n', result_nbr.sort_values(
```

Comment:
I can see the usefulness of print statements in tests, but it also seems like there should be a better way to get this information if we think it's needed by everyone (eg. a dedicated test that fails if a particular result is unexpected).  If this was just a debug print though, I think it should be removed.